web-app/src/utilities/AxiosInstance.js

```diff
@@ -55,6 +55,19 @@ instance.interceptors.response.use(function(response) {
     };
   }
 
+  if (response.config.url.includes(`${baseURL}/dashboard/getTeamEvents`)) {
```

Comment:
Is this axios middleware required? Does the _formatEventsForCalendar method in dashBoardEvents.js not convert the dates to moments too?

Reply:
Good spot. I think the issue here is actually in the dashboardEvents. I'd rather keep the place we transform dates into moments in the AxiosInstance in case the call was ever used elsewhere in the app. Just keeps it consistent too. 

Pushed a fix.